docs/troubleshooting.md

```diff
@@ -9,3 +9,7 @@ If you work on linux and cannot edit some of the project files right after the f
 If you have a SSL trust issues, download the self-signed certificate and run :
 
     $ sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain /path/to/you/certificate.cer
+
+## HTTPs and Redirects
+When Symfony is generating an internal redirect, and you're browsing via `https://...` you're redirected to `http://`.
```

Comment:
Can you put a link to the symfony-docs like :
```
[internal redirect](https://symfony.com/doc/current/routing.html#redirecting-to-urls-and-routes-directly-from-a-route)
```
And remove the link to the github issue please

Reply:
@maxhelias I reworderd that paragraph and added the link to the Symfony docs